### ISSUE TITLE
Accessibility - html tags need 'lang' attributes specifying English

### ISSUE THREAD

**Issue by RussellJQA** [RussellJQA/KevinCookieCompany.com#1]
Since each Web page is written in English, it needs an English `lang` attribute added:
`lang="en"`
to its existing html tag:
`<html>`
making it:
`<html lang="en">`

This is a failure of WCAG 2.2 [Success Criterion 3.1.1: Language of Page](https://www.w3.org/WAI/WCAG22/Understanding/language-of-page.html) -- a Level A ("must have") criterion.

[WCAG 2.2 is the current international Web accessibility standard.]
